src/libraries/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.1;netstandard2.0</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <!-- This library has unaddressed trim warnings in ILLink.Suppressions.xml -->
+    <EnableTrimAnalyzer>false</EnableTrimAnalyzer>
```

Comment:
Would it be better to simply detect the presense of `ILLink.Supressions.xml` and enable/disable analyzer accordingly?

Reply:
Yes, I plan to do that but didn't get there yesterday. I'm still trying to understand how the trimming tests are set up.